### ISSUE TITLE
Deity Locked when trying to respec a companion

### ISSUE THREAD

**Issue by Leohoshi** [BarleyFlour/RespecMod#179]
When trying to respec a companion from game you can't select the Deity he had previously already (I tried deleting the deity feat with toybox but the result was the same)

Likewise:

You can't take Shelyn if you respec Sosiel
You can't take Iomedae if you respec Lann or Seelah
You can't take Atheism if you respec Ember or Daeran (Did they find God? :O)
You can't take Callistria if you respec Wolijif
You can't take Desna if you respec Arueshalae
You can't take Green Faith if you respec Camellia
You can't take Godclaw if you respec Regill

I tried using the preserve function and it did work initially like 2 days ago, but the new version of the mod made the deities slot blank when trying to preserve it becomes a blank space and even if i create a new game, the companions come with Blank Deities already.

So the 'preserve' function is not working right now, if you try to respec preserving Background and Deity will be a blank at the character sheet. (I think it's just graphical issue, they actually keep the traits, but a very annoying graphical issue).

**Comment by Dark0Zero** [comment]
Are you using a mod that adds other deities? For example the Expanded Content mod breaks repec mods due to the changes it makes to how deities are selected, in this case by grouping them into categories that then have to be selected from.

**Comment by Leohoshi** [comment]
Yes I use Expanded Content, but the curious thing is the respec on main or custom companions characters works perfectly despite using Expanded Content Mod, my biggest and only problem are the game companions! it works on them but the deity problem happens which doesnt happen on custom characters.

**Comment by Dark0Zero** [comment]
Yep, that's the same problem I've had with it as well, and also why I don't use it anymore. The benefits just weren't worth the headaches. If I was using a full merc party it'd be different.

I can't really help with it, but you could try using Toybox to enable the conversational bits for companions who aren't in the party and just make a fully custom party. I've seen some people talking about making proxies for the regular companions via the mercs and then somehow making them look like the companions via the visual changes mod. I don't know how close you can get with this method though.

**Comment by hrjharry** [comment]
Is there a way to split background and deity choosing icon? 
Just find an already-known issue for deity choose when working with expanded content. So I wonder if it is possible to just choose a new background while not choosing the deity. A little weird to see these two are always combined together.
